storagenode/piecestore/endpoint_test.go

```diff
@@ -368,13 +368,15 @@ func GenerateOrderLimit(t *testing.T, satellite storj.NodeID, uplink storj.NodeI
 	require.NoError(t, err)
 	oe, err := ptypes.TimestampProto(time.Now().Add(orderExpiration))
 	require.NoError(t, err)
+
 	orderLimit := &pb.OrderLimit{
 		SatelliteId:     satellite,
 		UplinkId:        uplink,
 		StorageNodeId:   storageNode,
 		PieceId:         pieceID,
 		Action:          action,
 		SerialNumber:    serialNumber,
+		OrderCreation:	 time.Now().UTC(),
```

Comment:
I think `UTC()` here would have no real effect on how the `pb.OrderLimit` is serialized.